contracts/TracerPerpetualSwaps.sol

```diff
@@ -696,7 +688,7 @@ contract TracerPerpetualSwaps is
     modifier onlyLiquidation() {
         require(
             msg.sender == liquidationContract,
-            "TCR: Sender not liquidation contract "
+            "TCR: Sender not liquidation"
         );
         _;
```

Comment:
I feel like this makes it less clear but not too much of an issue

Reply:
Yeah valid @BenjaminPatch , was just doing this for gas optimisation. "Require strings
If we are adding message strings to require statements, we can make them cheaper by limiting the string length to 32 bytes." https://medium.com/coinmonks/gas-optimization-in-solidity-part-i-variables-9d5775e43dde